common/build.gradle.kts

```diff
@@ -29,4 +29,6 @@ dependencies {
   implementations(*Libs.coil)
   debugImplementations(Libs.chuckerDebug)
   releaseImplementations(Libs.chuckerRelease)
+
+  kaptTests(Libs.moshiCompiler)
```

Comment:
https://stackoverflow.com/questions/37926188/how-to-use-kapt-in-androidtest-scope